src/main/webapp/scripts/itinerary-overview.js

```diff
@@ -79,20 +104,57 @@ function geocodePlaceId(hotelID, placeIDArray) {
         });
     }
 
-    geocoder.geocode({placeId: hotelID}, function(results, status) {
+    geocoder.geocode({ placeId: hotelID }, (results, status) => {
         if (status === "OK") {
             if (results[0]) {
-                hotel += (results[0].geometry.location);
+                console.log("Hotel Address: " + results[0].formatted_address)
+                storeHotelAddress(results[0].formatted_address)
             } else {
                 window.alert("No results found");
             }
         } else {
             window.alert("Geocoder failed due to: " + status);
         }
     });
+}
+
 
-    console.log("Done Converting Place ID's");
-    calculateRoute(hotel, waypointsArray);
+/**
+Used to store the waypoint addresses given from geocodePlaceID()  in an Array to avoid from unordered execution
+ within the geocode function. Calls isReady() to test if the Array is set up to be used in calculateRoute() .
+ */
```

Comment:
JSDoc should have params annotated as well (this and all other JSDoc comments in this file)